src/plots/plots.js

```diff
@@ -1979,8 +1981,16 @@ plots.doAutoMargin = function(gd) {
 
     var gs = fullLayout._size;
     var margin = fullLayout.margin;
+    var reservedMargins = {'t': 0, 'b': 0, 'l': 0, 'r': 0};
```

Comment:
```suggestion
    var reservedMargins = {t: 0, b: 0, l: 0, r: 0};
```
we don't have a linter that enforces this?

Reply:
Enforced by #6513.

Reply:
Amazing! I made this mistake on a previous PR and it left me stuck for a while...